### PR TITLE
Add rubocop-rspec to the setup guide

### PR DESCRIPTION
**Pull request by coorasse**

### CODE REVIEW

File: ruby_on_rails/rspec.md [hunk 1]
```diff
@@ -58,30 +57,24 @@ Kernel.srand config.seed
...
 ```ruby
 require 'capybara/rspec'
 require 'capybara/rails'
 require 'selenium/webdriver'
 
-Capybara.register_driver :chrome do |app|
-  Capybara::Selenium::Driver.new(app, browser: :chrome)
-end
+RSpec.configure do |config|
+ # other configs
```
Comment (macav) [inline ruby_on_rails/rspec.md]: is this comment needed?

Comment (macav) [reply ruby_on_rails/rspec.md]: and is `config.use_transactional_fixtures = true` the default?

Comment (coorasse) [reply ruby_on_rails/rspec.md]: right! I added it to the rails_helper.rb template

Comment (coorasse) [reply ruby_on_rails/rspec.md]: for the comment: I wanted to show that the previous configurations are there, but is not in the template

File: templates/spec/rails_helper.rb [hunk 2]
```diff
@@ -4,38 +4,23 @@
...
-      example.run
-    end
+  config.before(:each, type: :system, js: true) do
+    driven_by :selenium_chrome_headless
   end
-
-  config.include FactoryBot::Syntax::Methods
-
-  config.infer_spec_type_from_file_location!
 end
```
Comment (schmijos) [inline templates/spec/rails_helper.rb]: It's kind of duplicated information. We have it in the rspec guide and here.

Comment (coorasse) [reply templates/spec/rails_helper.rb]: Indeed, but I like to see, while following the setup guide, what I need to add, and at the same time a copy of the final version of the file as a reference.

**Pull request merged**